src/pages/code-insights.tsx

```diff
@@ -60,6 +61,11 @@ const items = [
         title: 'Visualize configurations and services',
         text: <CodeInsightExample type={CodeInsightExampleType.Search} data={REPOS_WITH_CI_SYSTEM} />,
     },
+    {
+        title: 'Vulnerable Versions of OpenSSL in the Python Ecosystem',
+        text: <CodeInsightExample type={CodeInsightExampleType.Search} data={OPENSSL_PYTHON} />,
+    },
```

Comment:
I think we should make this 4th in the list to showcase it a little higher, what do you think?

Reply:
SGTM!